www/IndoorAtlas.js

```diff
@@ -95,7 +95,8 @@ function IndoorAtlas() {
   // none of the Cordova callbacks are supposed to fail, always report
   // as status OUT_OF_SERVICE if something like this happens
   function error(result) {
-    var message = 'internal error' + methodName + ' failed ' + JSON.stringify(result);
+    //var message = 'internal error' + methodName + ' failed ' + JSON.stringify(result);
+    var message = 'internal error ' + JSON.stringify(result);
```

Comment:
You're right, this line is the correct version. We'll merge https://github.com/IndoorAtlas/cordova-plugin/pull/37/commits/5029fc57ae65e576f436ed0aa5c5971867763ab4 in the near future. Thanks again!